test/version/version.test.js

```diff
@@ -140,17 +139,6 @@ describe('single version flag', () => {
         expect(stdout).toContain(`webpack-dev-server ${webpackDevServerPkgJSON.version}`);
     });
 
-    it('outputs version with migrate', () => {
```

Comment:
Keep it, just simplify version checking

Reply:
then we need to keep `packages/migrate/package.json` is that fine?

Reply:
need to keep package.json file of migrate package, should we keep it and README and write a deprecation message there?

Reply:
No, just remove version checking

Reply:
since it's removed locally it won't show up in help, will write a separate test for this

Reply:
It will work, we use lazy loading commands, so when you run help, command try to load all command what exists and uses them in help

Reply:
i.e. if you have `@webpack-cli/migrate` inside `node_modules` it will be in help

Reply:
fixed

Reply:
lazy loading is brilliant, should also boost perf greatly 💯